src/herder/test/HerderTests.cpp

```diff
@@ -1613,16 +1607,16 @@ TEST_CASE("SCP Driver", "[herder][acceptance]")
 {
     SECTION("protocol 10")
     {
-        testSCPDriver(10, 10, 10, false, true, false);
+        testSCPDriver(10, 10, 10, false, false);
```

Comment:
this test should have been removed in the previous commit

Reply:
Removed.